### PR TITLE
Add Rancher scheduling labels

### PR DESCRIPTION
**Pull request by adamgraves-choices**
Enables Rancher scheduling, so containers should deployed onto hosts with the label "c2s=true" before any other host.

Part of the work required for: https://trello.com/c/loOXIE1j/1-multi-region-provider-hosting

### CODE REVIEW

File: templates/c2s-pharmacy-finder/1/rancher-compose.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@ version: '2'
 
 catalog:
   name: nhsuk_c2s
```
Comment (beseven) [inline templates/c2s-pharmacy-finder/1/rancher-compose.yml]: Can you remove the ^M - https://github.com/nhsuk/nhsuk-rancher-templates/pull/13/commits/1f97e9d75579968536bca8dec240ae322c74cd70

**Review by beseven — CHANGES_REQUESTED** [review]
How does having the directory folder impact on the ci-deployments scipts at the moment? @neilbmclaughlin @st3v3nhunt

File: templates/c2s-pharmacy-finder/1/rancher-compose.yml (hunk 1 above)
Comment (adamgraves-choices) [reply templates/c2s-pharmacy-finder/1/rancher-compose.yml]: The invalid chars were added previously. I've removed them locally but the changes haven't been pushed for some reason.

**Comment by adamgraves-choices** [comment]
The CI-deployment scripts will still deploy the old version, but then again, the gp-pharmacy-finders template already has a `1/` directory, so that must be currently deploying the old version.

**Review by beseven — APPROVED** [review]

**Pull request merged**